modules/integration/src/test/scala/scala/cli/integration/RunTestDefinitions.scala

```diff
@@ -1234,4 +1234,21 @@ abstract class RunTestDefinitions(val scalaVersionOpt: Option[String])
       expect(p.out.text().trim == "hello")
     }
   }
+
+  test("workspace dir") {
+    val inputs = TestInputs(
+      Seq(
+        os.rel / "Hello.scala" ->
+          """|// using lib com.lihaoyi::os-lib:0.7.8
+             |
+             |object Hello extends App {
+             |  println(os.pwd) 
+             |}""".stripMargin
+      )
+    )
+    inputs.fromRoot { root =>
+      val p = os.proc(TestUtil.cli, "Hello.scala").call(cwd = root)
```

Comment:
I'm not sure if this test really tests the code that has been change. When you run `scala-cli run ... ` it takes the cwd from `Inputs`, which just takes os.cwd `https://github.com/VirtusLab/scala-cli/blob/75b04cc08d265423cd608f3cebc75d18ba181dd4/modules/cli/src/main/scala/scala/cli/commands/SharedOptions.scala#L218`

So it passes even without the change. In order to test it, you need to run `scala-cli bsp`